### ISSUE TITLE
heatshrinkpy cannot be used properly.

### ISSUE THREAD

**Issue by banana-sun** [yawor/heatshrinkpy#1]
OS : WIN 10
python : 3.10.7

When I execute _pip install heatshrinkpy_ after the normal installation is complete, an error occurs when executing with the following code.

```
from heatshrinkpy.core import decode, encode

str=None
with open("src.txt", 'rb') as src:
    str = src.read()

bin = encode(str, window_sz2=12, lookahead_sz2=4)
debin = decode(bin, window_sz2=12, lookahead_sz2=4)

print(f"{bin.hex(' ')}, {debin}")
```
Error message:

> Traceback (most recent call last):
  File "...\test.py", line 1, in <module>
    from heatshrinkpy.core import decode, encode
  File "...\Python\Python310\lib\site-packages\heatshrinkpy\__init__.py", line 5, in <module>
    from . import core
ImportError: cannot import name 'core' from partially initialized module 'heatshrinkpy' (most likely due to a circular import) (...\Python\Python310\lib\site-packages\heatshrinkpy\__init__.py)

After checking, the core folder was lost during the installation process, resulting in unavailability. Manually copy the core folder to the path and it is ready to work. I hope this problem can be solved. Thank you.

**Comment by yawor** [comment]
Thanks for the report. I've migrated the package from setuptools to poetry and I've published new version with your fix from #2. Please check if this fixes your problem on Windows.

**Comment by banana-sun** [comment]
It's perfect with the new version.
